### ISSUE TITLE
Puzzle piece 

### ISSUE THREAD

**Issue by adjmiahmed** [dragosholban/AndroidPuzzleGame#3]
Hi i want to be able to put a curve in the bottom and rigth side. but with ur code its not possible is there any thing i can change to be able to use the curve on the bottom and rigth (what i mean by curve is that a cercle out not in like this ( .
thanks for ur time :)

**Comment by StefanOltmann** [comment]
Yeah, I tried it, but this is were it get's quickly pretty complicated with the offset of the bitmap.

Basicly what you have to do is applying that "offset" to all sides (not just "top" and "left").
After that you have to handle that the bitmap is now larger and adjust all calculations based on that.

I cancelled my fist attempt of doing that because I got lost in the calculation logic.
After some refactorings I may try it again.
